sorc/link_workflow.sh

```diff
@@ -215,10 +215,12 @@ fi
 if [[ -d "${HOMEgfs}/sorc/gsi_monitor.fd" ]]; then
 
   cd "${HOMEgfs}/fix" || exit 1
-  [[ ! -d gdas ]] && ( mkdir -p gdas || exit 1 )
-  cd gdas || exit 1
+  [[ ! -d mon ]] && ( mkdir -p mon || exit 1 )
+  cd mon || exit 1
   ${LINK_OR_COPY} "${HOMEgfs}/sorc/gsi_monitor.fd/src/Minimization_Monitor/nwprod/gdas/fix/gdas_minmon_cost.txt" .
   ${LINK_OR_COPY} "${HOMEgfs}/sorc/gsi_monitor.fd/src/Minimization_Monitor/nwprod/gdas/fix/gdas_minmon_gnorm.txt" .
+  ${LINK_OR_COPY} "${HOMEgfs}/sorc/gsi_monitor.fd/src/Minimization_Monitor/nwprod/gfs/fix/gfs_minmon_cost.txt" .
+  ${LINK_OR_COPY} "${HOMEgfs}/sorc/gsi_monitor.fd/src/Minimization_Monitor/nwprod/gfs/fix/gfs_minmon_gnorm.txt" .
```

Comment:
These are just text files. Shouldn't they be in `parm/`?

Reply:
Moving to `parm/monitor`.